### ISSUE TITLE
Running Root Command tkn Shouldn't Give Error Message

### ISSUE THREAD

**Issue by danielhelfand** [tektoncd/cli#139]
Currently, running just running `tkn` responds to the user with an error message: `Error: Use one of available subcommands: completion [SHELL], help [command], pipeline, pipelinerun, task, taskrun, version`. It would be better presentation to respond with a description of `tkn`, the version of `tkn`, and the list of available commands for `tkn`. 

# Expected Behavior
Run `tkn` and get the following:

1. A note that this is the Tekton Pipelines cli 
2. The version of the CLI
3. Standard `--help` information like you already have:
```
Usage:
  tkn [flags]
  tkn [command]

Available Commands:
  completion  Prints shell completion scripts
  help        Help about any command
  pipeline    Manage pipelines
  pipelinerun Manage pipelineruns
  task        Manage tasks
  taskrun     Manage taskruns
  version     Prints version information

Flags:
      --azure-container-registry-config string   Path to the file containing Azure container registry configuration information.
  -h, --help                                     help for tkn

Use "tkn [command] --help" for more information about a command.
```

# Actual Behavior

```
Error: Use one of available subcommands: completion [SHELL], help [command], pipeline, pipelinerun, task, taskrun, version
Usage:
  tkn [flags]
  tkn [command]

Available Commands:
  completion  Prints shell completion scripts
  help        Help about any command
  pipeline    Manage pipelines
  pipelinerun Manage pipelineruns
  task        Manage tasks
  taskrun     Manage taskruns
  version     Prints version information

Flags:
      --azure-container-registry-config string   Path to the file containing Azure container registry configuration information.
  -h, --help                                     help for tkn

Use "tkn [command] --help" for more information about a command.
```

# Steps to Reproduce the Problem

1. Run `tkn` using `tkn Client version: 0.1.2`

# Additional Info

Using `Client version: 0.1.2` on macOS. Downloaded via `brew`.

**Comment by chmouel** [comment]
This should be fixed with my latest commit in 1c270872a047ece618b117c060e6d34a57f064c1 : 

<img width="1325" alt="image" src="https://user-images.githubusercontent.com/98980/60567647-e3799100-9d6a-11e9-8e2c-5874d86fbf96.png">

/close

**Comment by danielhelfand** [comment]
@chmouel Thanks! Looks good.
